.scons_localrc

```diff
@@ -0,0 +1,11 @@
+#
```

Comment:
I think we do not need to do this if we just do
module load mvapich2-x/2.3
priori to build and test

Reply:
build_env.sh essentially does this, except it just manually sets the environment variables. (I also tried with just modue load mvapich2-x/2.3).
The problem I ran into was that scons doesn't necessarily inherit the environment that it is called from. mvapich2 uses the icc compiler, which is not on the standard paths (i.e. /usr/lib, /usr/bin). scons (daos build) seems to just look for the location of mpicc, or gets this from MPI_PKG. But either way, the icc binary and libraries aren't included in the scons build environment.
Maybe there is an easier way, but without modifying daos itself, I couldn't get it to work.

Reply:
Try this. It works for me.
module unload intel/19.1.1
module load gcc/9.1.0
module load mvapich2-x/2.3